config/account.go

```diff
@@ -341,6 +341,7 @@ type AccountPrivacy struct {
 }
 
 type PrivacySandbox struct {
+	TopicsDomain      string            `mapstructure:"topicsdomain"`
```

Comment:
Please add an entry in `SetupViper` to enable environment variable configuration. Can either be:

```
v.SetDefault("account_defaults.privacy.privacysandbox.topicsdomain", "")
```

or 

```
v.BindEnv("account_defaults.privacy.privacysandbox.topicsdomain")
```

Reply:
done